### PR TITLE
Added "display: inline-block;" for a.sbtn

### PR DESCRIPTION
**Pull request by adityasinghbaghel**
Issue:
#134

### CODE REVIEW

File: src/components/_basic.less [hunk 1]
```diff
@@ -35,6 +35,7 @@ a.sbtn {
   &:hover,
```
Comment (shahednasser) [inline src/components/_basic.less:35]: remove `&:hover, &:active` it should be on `a.sbtn` and it'll be applied for all states as well.

**Review by shahednasser — CHANGES_REQUESTED** [review]
You need to compile your code by running `npm run compile` in order for the changes to take effect.

File: src/components/_basic.less (hunk 1 above)
Comment (adityasinghbaghel) [reply src/components/_basic.less:35]: ok

**Review by shahednasser — CHANGES_REQUESTED** [review]
Are you sure you compiled the less files? If you ran the compile command which is `npm run compile`, `dist/sbuttons.min.css` would've changed too. Since it hasn't changed, no changes can be seen on the website. If you're having trouble with this, please check the [contribution guide](https://github.com/shahednasser/sbuttons/blob/master/CONTRIBUTING.md) for more details

**Comment by adityasinghbaghel** [comment]
Sorry , it was my mistake I forgot to run ( npm run watch)

**Comment by shahednasser** [comment]
thank you!

@all-contributors please add @adityasinghbaghel for code and bug

**Pull request merged**